test/api/profileImageUploadSpec.ts

```
@@ -18,6 +18,13 @@
     const form = frisby.formData()
     form.append('file', fs.createReadStream(file))
 
+
+  const apiKey = '12345-ABCDE';
```

ESLint

[error] More than 1 blank line not allowed.

(no-multiple-empty-lines)